lib/rototiller/task/beaker_task.rb

```diff
@@ -0,0 +1,74 @@
+require 'rake'
+require 'rototiller/task/flags/cli_flags'
+
+module Rototiller
+  module Task
+    class BeakerTask
```

Comment:
this seems like it could get most of its functionality from RototillerTask.  should it inherit?